zaza/openstack/charm_tests/vault/tests.py

```diff
@@ -257,6 +257,60 @@ def test_zzz_pause_resume(self):
         lead_client = vault_utils.extract_lead_unit_client(self.clients)
         self.assertTrue(lead_client.hvac_client.seal_status['sealed'])
 
+    def test_vault_reload(self):
+        """Run reload tests.
+
+        Reload service and check services were restarted
+        by doing simple change in the running config by API.
+        Then confirm that service is not sealed
+        """
+        vault_actions = zaza.model.get_actions(
+            'vault')
+        if 'reload' not in vault_actions:
```

Comment:
Since these are new tests that will only get run by the master charm-vault charm I don't think we need to test if the restart action exists.

Reply:
I have followed the same approach as for the rest of the tests